databricks-sdk-java/src/main/java/com/databricks/sdk/core/DefaultCredentialsProvider.java

```diff
@@ -111,6 +112,20 @@ private void addOIDCCredentialsProviders(DatabricksConfig config) {
     }
 
     List<NamedIDTokenSource> namedIdTokenSources = new ArrayList<>();
+    namedIdTokenSources.add(
+        new NamedIDTokenSource(
+            "env-oidc",
+            new EnvVarIDTokenSource(
+                // Use configured environment variable name if set, otherwise default to
+                // DATABRICKS_OIDC_TOKEN
+                (Strings.isNullOrEmpty(config.getOidcTokenEnv()))
+                    ? "DATABRICKS_OIDC_TOKEN"
+                    : config.getOidcTokenEnv(),
```

Comment:
Are these parentheses needed?

```suggestion
                Strings.isNullOrEmpty(config.getOidcTokenEnv())
                    ? "DATABRICKS_OIDC_TOKEN"
                    : config.getOidcTokenEnv(),
```